contracts/interfaces/IBridge.sol

```diff
@@ -0,0 +1,8 @@
+pragma solidity 0.6.4;
+
+interface IBridge {
+
+    // not sure if this should be done, exposing getter for _ChainID only
+    // better than using call
+    function _chainID() external returns (uint);
```

Comment:
Could you elaborate?

Reply:
Hi David, just wasn't sure if it was better to create a whole interface to expose a single function vs using a Call. I chose to use the interface for extensibility + i think sometimes using a Call is rather unintuitive in the code for people reviewing it